app/src/components/Card/CardPreview/index.tsx

```diff
@@ -49,87 +36,114 @@ function CardPreview({ card, onClick }: ICardPreviewProps) {
           </div>
         </div>
-        <div className="flex-shrink-0">
+        <div className='flex-shrink-0'>
           {card.title && (
-            <div className="h-[62px] w-full flex-col justify-start items-start gap-3 inline-flex">
-              <div className="self-stretch text-[#46434e] text-xl font-medium font-['Rubik'] leading-7 overflow-hidden" style={{ display: '-webkit-box', WebkitLineClamp: 2, WebkitBoxOrient: 'vertical', overflow: 'hidden', textOverflow: 'ellipsis' }}>
+            <div className='h-[62px] w-full flex-col justify-start items-start gap-3 inline-flex'>
+              <div
+                className="self-stretch text-[#46434e] text-xl font-medium font-['Rubik'] leading-7 overflow-hidden"
+                style={{
+                  display: '-webkit-box',
+                  WebkitLineClamp: 2,
+                  WebkitBoxOrient: 'vertical',
+                  overflow: 'hidden',
+                  textOverflow: 'ellipsis',
+                }}
+              >
                 {card.title}
               </div>
             </div>
           )}
         </div>
-        <div className="flex-grow overflow-hidden">
+        <div className='flex-grow overflow-hidden'>
           {!isEmpty(tasks) ? (
-            <div className="flex flex-col gap-3">
-              <div className="col-span-3 overflow-hidden">
-                {tasks.slice(0, 3).map((task) => (
-                  <div key={'prevtask-' + task.id} className="h-[46px] py-1 justify-start items-start gap-2 inline-flex">
-                    <div className="w-4 h-4 justify-center items-center flex">
-                      <div className="w-4 h-4 pl-[1.60px] pr-[1.23px] pt-[1.60px] pb-[1.30px] justify-center items-center inline-flex">
-                        <div className="w-[13.17px] h-[13.10px] relative">
-                          {/* Здесь можно вставить содержимое, например, иконку */}
-                        </div>
+            <div className='flex flex-col gap-3'>
+              <div className='col-span-3 overflow-hidden'>
+                {tasks.slice(0, MAX_TASKS).map((task) => (
+                  <div
+                    key={'prevtask-' + task.id}
+                    className='h-[46px] py-1 justify-start items-start gap-2 inline-flex'
+                  >
+                    <div className='w-4 h-4 justify-center items-center flex'>
+                      <div className='w-4 h-4 pl-[1.6px] pr-[1.23px] pt-[1.6px] pb-[1.3px] justify-center items-center inline-flex'>
+                        <i className='ri-checkbox-line text-14' />
                       </div>
                     </div>
-                    <div className="grow shrink basis-0">
-                      <span className="text-[#46434e] text-sm font-normal font-['Rubik'] leading-[18.90px]">
+                    <div className='grow shrink basis-0'>
+                      <span className="text-[#46434e] text-sm font-normal font-['Rubik'] leading-[18.9px]">
```

Comment:
Why are there single quotes in the first line, but there are double quotes in the second one 🤔I guess smth wrong with prettier again on your side. I will come back and set up eslint finally